config/routes.rb

```diff
@@ -4,6 +4,8 @@
   constraints format: 'html' do
     resources :categories, only: 'show', path: 'browse'
 
+    get '/book', to: redirect('/book-phone')
```

Comment:
This will create a 301 "Moved Permanently" redirect which we don't want. We should do this upstream where we can set a 302 redirect.